war/src/main/less/pluginSetupWizard.less

```diff
@@ -102,6 +101,11 @@
     margin-bottom: 1em;
   }
 
+  .danger {
+    border: 1px solid;
+    border-color: var(--danger-color, #c4000a);
+  }
```

Comment:
I see it does this:

<img width="872" alt="image" src="https://user-images.githubusercontent.com/21194782/188285000-51d68c0a-1687-48c2-8296-826f2cef9e3b.png">

seems unrelated to the PR?

Reply:
### Before
<img width="657" alt="Error Before" src="https://user-images.githubusercontent.com/71805759/188315185-ffd1727d-bfc8-4863-9117-6a24af788b4a.PNG">

### After
<img width="676" alt="Error After" src="https://user-images.githubusercontent.com/71805759/188315195-dd68eb53-90ce-4305-bbea-5b4370f76f55.PNG">

I attached the before and after screenshot, I found that it's use red border in the `Signin Page` when you enter wrong username or password (you can see it below), I'm not sure it's neccessary here or not, if it's unnecessary, i'll remove it later.



<img width="267" alt="Signin png" src="https://user-images.githubusercontent.com/71805759/188315218-be999f25-9fe2-4136-acc2-a6e0cae3f99f.PNG">